.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit =
```

Comment:
if we would limit to `frontend` and `backend` this file does not make sense here.